libraries/helpers.rb

```diff
@@ -9,7 +9,7 @@ def nagios_vname
         'nagios4'
       elsif platform?('ubuntu')
         case node['platform_version'].to_f
-        when 16.04, 18.04
+        when 18.04
```

Comment:
This whole thing could be improved to make this more future proof. Remove the case statement and just make it

```ruby
if node['platform_version'].to_f < 20.04
  'nagios3'
else 
  'nagios4'
end
```

Then it just magically works on 21.x